### PR TITLE
[ADD] business_requirement_crm

### PR DESCRIPTION
**Pull request by xie8899**

### CODE REVIEW

File: business_requirement_crm/models/business.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    _inherit = "crm.lead"
+
+    project_id = fields.Many2one(
+        comodel_name='project.project',
+        string='Project',
+        ondelete='set null',
+    )
+    resource_cost_total = fields.Float(
+        compute='_get_resource_cost_total',
+        string='Total Revenue from BR',
```
Comment (dreispt) [inline business_requirement_crm/models/business.py]: Revenue or Cost?

Comment (xie8899) [inline business_requirement_crm/models/business.py]: Revenue

Comment (elicoidal) [inline business_requirement_crm/models/business.py]: @xie8899 
this is confusing here. 
I agree this is the Revenue but the methods and names are confusing (you name them "resource").
you should have _get_deliverable_price_total
and resource_cost_total should be deliverable_price_total (please check my comment on PR#138)
in general use resource and cost on one side and deliverable and price on the other side.

File: business_requirement_crm/README.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: https://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================
+Business Requirement Crm
```
Comment (elicoidal) [inline business_requirement_crm/README.rst]: s/Crm/CRM

File: business_requirement_crm/README.rst [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+========================
+
+This module is part of a set ("Business Requirement") and extend the basic functionality to link the business requirement to CRM opportunity,
```
Comment (elicoidal) [inline business_requirement_crm/README.rst]: Make sure to respect the 80 char for descriptions

**Comment by elicoidal** [comment]
@jbeficent @dreispt This PR starts to be mature. I have tested it functionally and it could still be improve but I think it is usable now.
it is linked to PR 127 (deliverable - mature), 138 (deliverable - mature), 144 (crm - stable but improving).
We should push a PR today.

**Comment by elicoidal** [comment]
replaced by https://github.com/OCA/business-requirement/pull/8

**Pull request closed without merging**